packages/services/src/index.ts

```diff
@@ -19,4 +19,5 @@ export * from "./simulated-trade-execution.service.js";
 export * from "./trade-simulator.service.js";
 export * from "./trading-constraints.service.js";
 export * from "./user.service.js";
+export * from "./utils/pagination.js";
```

Comment:
I kept this top level index.ts just for exporting services. There are other entrypoints in subdirectories, including `lib`. Just add this there then callers can access it as `import { buildPaginationResponse } from "@recallnet/services/lib`, for example.

Reply:
And also this needs to change before approval.